modules/test-runner/src/createChannel/createChannel.test.ts

```diff
@@ -10,6 +11,16 @@ import {
 } from "../util";
 
 describe("Create Channel", () => {
+  it("Happy case: user creates channel with mainnet network string", async () => {
+    const clientA: IConnextClient = await createDefaultClient("mainnet");
+    expect(clientA.multisigAddress).to.be.ok;
```

Comment:
we should also add a test that if you instantiate the default client and a client with client opts you get the same multisig address